### PR TITLE
Add params

### PR DESCRIPTION
**Pull request by aminamos**

### CODE REVIEW

File: src/video/domain.ts [hunk 1]
```diff
@@ -251,6 +262,9 @@ export interface CreateLiveStreamParams {
   latency_mode?: LatencyMode;
   simulcast_targets?: Array<SimulcastTargetParams>;
   test?: boolean;
+  audio_only?: boolean;
```
Comment (jaredsmith) [inline src/video/domain.ts:265]: Won't this conflict with #170 ?

**Review by jaredsmith — COMMENTED** [review]
What has been done to test this?

File: src/video/domain.ts (hunk 1 above)
Comment (aminamos) [reply src/video/domain.ts:265]: If #170 is merged first, I don't think so?

Comment (aminamos) [reply src/video/domain.ts:265]: I'll pull again just in case

**Comment by aminamos** [comment]
> What has been done to test this?

I ran a local test but can add the actual tests to the directory now

**Pull request merged**